core/src/App.html

```diff
@@ -30,7 +46,20 @@
         handleRouteChange(trimLeadingSlash(window.location.pathname), component, node, config);
     };
 
-    const handleNavigation = async (component, data) => {
+    const handlePreserveView = (component, data, config, node) => {
+        const pv = component.get().preservedViews;
+        if (data.params.preserveView) {
+            pv.push({
+                path: getNodePath(component.get().currentNode),
+                nextPath: getNodePath(component.get().currentNode) + data.params.link,
+                context: component.get().context
+            });
+            component.set({preservedViews: pv});
+            config.iframe = null;
```

Comment:
Please write a comment specifying we need to set this here so that a new iframe will be created

Reply:
done